examples/simple-lambda-with-deploy-test/example.tf

```diff
@@ -1,29 +1,28 @@
 provider "aws" {
-  version = "~> 2.56"
+  version = "~> 3.0"
   region  = "us-west-2"
 }
 
 module "acs" {
-  source = "github.com/byu-oit/terraform-aws-acs-info?ref=v2.1.0"
+  source = "github.com/byu-oit/terraform-aws-acs-info?ref=v3.4.0"
 }
 
 module "lambda_api" {
-  # source                        = "../../"
-  source                        = "github.com/byu-oit/terraform-aws-lambda-api?ref=v1.3.0"
-  app_name                      = "my-lambda-codedeploy"
-  env                           = "dev"
-  codedeploy_service_role_arn   = module.acs.power_builder_role.arn
-  lambda_zip_file               = "./src/lambda.zip"
-  handler                       = "index.handler"
-  runtime                       = "nodejs12.x"
+  #  source = "../../"
```

Comment:
I'm just tagging these fmt things when I see them. I might be off on if these will pass or not, so if I am feel free to ignore

Reply:
This is passing `terraform fmt -check` and it's also what the `terraform fmt` did